resources/views/components/card/address.blade.php

```diff
@@ -0,0 +1,27 @@
+@props(['shipping' => false, 'billing' => false])
+
+<address-card
+    {{ $attributes->whereStartsWith('v-') }}
+    :shipping="{{ var_export($shipping) }}"
+    :billing="{{ var_export($billing) }}"
+    v-slot="{ address, billing, shipping, isEmpty }"
+>
+    <x-rapidez-ct::card {{ $attributes->whereDoesntStartWith('v-') }} v-if="!isEmpty">
+        <x-rapidez-ct::title.lg class="mb-4">
+            <template v-if="billing && shipping">@lang('Shipping & billing address')</template>
+            <template v-else-if="shipping">@lang('Shipping address')</template>
+            <template v-else-if="billing">@lang('Billing address')</template>
+            <template v-else>@lang('Address')</template>
+        </x-rapidez-ct::title.lg>
+        <div class="flex flex-wrap justify-between">
+            <ul class="[&>*+*]:mt-1">
```

Comment:
`space-y-*` or `gap-y-*` is maybe cleaner?

Reply:
@jordythevulder